### PR TITLE
Added bilinear and nearest interp v2 oneDNN FP32 kernels

### PR DESCRIPTION
**Pull request by jakpiase**
### PR types
New features

### PR changes
OPs

### Describe
Added oneDNN FP32 kernels for nearest_interp_v2 and bilinear_interp_v2

### CODE REVIEW

**Comment by jakpiase** [comment]
@luotao1 Could you please review this code?

**Comment by jakpiase** [comment]
@lidanqing-intel @jczaja @wozna Could you please review this PR?

File: paddle/fluid/operators/mkldnn/interpolate_v2_mkldnn_op.cc [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        out_dims = out_size_data;
+      }
+    } else {
+      std::vector<float> scales;
+      auto scale_tensor = ctx.Input<Tensor>("Scale");
+      if (scale_tensor != nullptr) {
+        auto scale_data = get_new_data_from_tensor<float>(scale_tensor);
+        std::copy(scale_data.begin(), scale_data.end(), scales.begin());
+        while (scale_data.size() < 3) scales.push_back(scale_data[0]);
+      } else {
```
Comment (lidanqing-vv) [inline paddle/fluid/operators/mkldnn/interpolate_v2_mkldnn_op.cc]: Hi, Is this else branch only part that is different from interpolate_mkldnn_v1?
Maybe function `std::is_same` for type comparison could be used for this case and we could combine interpolate v1 and v2?

File: python/paddle/fluid/tests/unittests/mkldnn/test_bilinear_interp_v2_mkldnn_op.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (lidanqing-vv) [inline python/paddle/fluid/tests/unittests/mkldnn/test_bilinear_interp_v2_mkldnn_op.py]: year 2021

**Comment by lidanqing-vv** [comment]
@jczaja @wozna  Hi, if you have time, could you please also review this PR? We need 1 more review I guess

**Comment by lidanqing-vv** [comment]
LGTM

**Review by jczaja — DISMISSED** [review]
LGTM

**Comment by jakpiase** [comment]
@luotao1 Hi, could you please start your review?

**Review by jczaja — APPROVED** [review]

**Review by luotao1 — APPROVED** [review]

**Pull request merged**